kura/org.eclipse.kura.web2/src/main/resources/org/eclipse/kura/web/client/messages/Messages.properties

```diff
@@ -315,9 +315,8 @@ netIPv4Address=IP Address
 netIPv4SubnetMask=Subnet Mask
 netIPv4Gateway=Gateway
 netIPv4DNSServers=DNS Servers
-netIPv4SearchDomains=Search Domains
 netIPv4InvalidPriority=Priority allowed values are only integers from -1 to 2147483647.
-netIPv4InvalidAddress=Invalid IPv4 Address
+netIPv4InvalidAddress=Invalid IP Address
```

Comment:
Is this used also for IPv6? If so, maybe it is better to add a new message entry...

Reply:
I didn't want to replicate the same message twice for IPv4 and IPv6

Reply:
Latest commit separates better the two cases.